hoodie-common/src/main/java/com/uber/hoodie/common/util/CompactionUtils.java

```diff
@@ -97,4 +102,53 @@ public static HoodieCompactionOperation buildHoodieCompactionOperation(Compactio
   public static CompactionOperation buildCompactionOperation(HoodieCompactionOperation hc) {
     return CompactionOperation.convertFromAvroRecordInstance(hc);
   }
+
+  /**
+   * Get all pending compaction workloads along with their instants
+   *
+   * @param metaClient Hoodie Meta Client
+   */
+  public static List<Pair<HoodieInstant, HoodieCompactionWorkload>> getAllPendingCompactionWorkloads(
```

Comment:
is there a test for this file? I think we should add tons of tests around these methods in this file

Reply:
Unit tests added